src/main/java/core/basesyntax/Data.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Data {
```

Comment:
It is a really good example of a bad naming)))
What kind of data? What i should use it with? Why does it have a method draw? And other questions can appear in your colleague's brain) Take pity on them out)

Reply:
😭